pages.bn/android/dumpsys.md

```diff
@@ -0,0 +1,29 @@
+# dumpsys
+
+> অ্যান্ড্রয়েড সিস্টেম পরিষেবা সম্পর্কে তথ্য প্রদান করে।
+> এই কমান্ডটি শুধুমাত্র `adb shell` এর মাধ্যমে ব্যবহার করা যেতে পারে।
+> আরও তথ্য পাবেন: <https://developer.android.com/studio/command-line/dumpsys>.
+
+- সমস্ত সিস্টেম পরিষেবার জন্য ডায়াগনস্টিক আউটপুট পান:
+
+`dumpsys`
+
+- একটি নির্দিষ্ট সিস্টেম পরিষেবার জন্য ডায়গনিস্টিক আউটপুট পান:
+
+`dumpsys {{পরিষেবা}}`
+
+- 'ডাম্পসিস' যে সমস্ত পরিষেবা সম্পর্কে তথ্য দিতে পারে তাদের তালিকা করুন:
+
+`dumpsys -l`
+
+- একটি পরিষেবার জন্য পরিষেবা-নির্দিষ্ট আর্গুমেন্ট তালিকা করুন:
+
+`dumpsys {{পরিষেবা}} -h`
+
+- ডায়গনিস্টিক আউটপুট থেকে একটি নির্দিষ্ট পরিষেবা বাদ দিন:
+
+`dumpsys -- skip {{পরিষেবা}}`
+
+- সেকেন্ডে একটি সময়সীমা নির্দিষ্ট করুন (ডিফল্ট 10 সেকেন্ডে):
+
+`dumpsys -t {{সেকেন্ড}}`
```

Comment:
```suggestion
- সেকেন্ডে একটি সময়সীমা নির্দিষ্ট করুন (ডিফল্ট 10 সেকেন্ডে):

`dumpsys -t {{1..infinity}}`
```

Optional fix (error source: original page): range syntax can be used.
Warning: no explicitly valid number range is specified in the documentation, `1..infinity` was used as a fallback.

Reply:
Should I translate the number when using ranged syntax?

Reply:
It's your wish to translate the number if required. I would suggest something like 1...অনন্ত